### DISCUSSION TITLE
The call to the api returns an empty array

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#12630] [Questions] [answered]
Hi, I'm making a new app. I went through the initial setup process in `React` by creating the client for supabase. The problem that is appearing to me is that I am making a call to return the values ​​of a table and it returns `[]`. In the client configuration I have the variables `url` and `anon-key` set

```
import { createClient } from '@supabase/supabase-js';

const supabaseUrl = process.env.REACT_APP_SUPABASE_URL;
const supabaseAnonKey = process.env.REACT_APP_SUPABASE_ANON_KEY;

export const supabase = createClient(supabaseUrl, supabaseAnonKey)
```

The calling function is configured this way.

```
  const getSpecialties = async() => {
    const { data, error } = await supabase
    .from('roles')
    .select('*')
    console.log(data)
    console.log(error)
  }
```

what returns me: `Array []`

**Comment by GaryAustin1** [comment, marked as answer]
Usually that means your RLS is enabled and you have no policies or you do not meet the policies.

**Reply by ricardosikic** [reply]
thanks it works
